environment.yml

```diff
@@ -7,27 +7,30 @@ channels:
   - bioconda
   - defaults
 dependencies:
+  - conda-forge::openblas=0.3.7
   - bioconda::adapterremoval=2.3.1
-  - bioconda::bcftools=1.9
-  - bioconda::bedtools=2.29.0
-  - conda-forge::bokeh=1.3.4
-  - bioconda::bowtie2=2.3.5
-  - bioconda::damageprofiler=0.4.6
-  - bioconda::fastqc=0.11.8
+  - bioconda::bcftools=1.10.2
+  - bioconda::bedtools=2.29.2
+  - bioconda::blast=2.9.0
+  - conda-forge::bokeh=2.0.1
+  - bioconda::bowtie2=2.3.5.1
+  - bioconda::damageprofiler=0.4.9
+  - bioconda::fastqc=0.11.9
   - conda-forge::jupyter=1.0.0
   - bioconda::kraken2=2.0.8_beta
-  - conda-forge::notebook=6.0.1
+  - conda-forge::notebook=6.0.3
   - conda-forge::nbconvert=5.6.1
-  - bioconda::nextflow=19.10.0
+  - bioconda::nextflow=20.01.0
```

Comment:
Why do you bundle nextflow inside coproids container?

Reply:
Removed it